app/views/layouts/application.html.erb

```diff
@@ -11,6 +11,7 @@
   </head>
 
   <body>
+    <%= render partial: "pages/shared/navbar" %>
     <%= yield %>
   </body>
```

Comment:
You removed some code that was part of the master branch.
You need to add itback to resolve your conflict

Reply:
As for your render, can you wrap it inside a `<header>`  tag ?
And you don't need to write partials. `<%= render "shared/navbar" %>` is enough